### PR TITLE
completed 7

### PR DESCRIPTION
**Pull request by steelfyi**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -173,9 +173,120 @@ describe("200 - GET: /api/articles/:article_id/comments", () => {
...
       .then((response) => {
-        console.log(response.body);
         expect(response.body).toEqual({ comments: [] });
-        // expect(response.body).toEqual({ msg: "Comment not found" });
       });
   });
 });
+
+// YOUR TESTS
+describe("200 - POST: /api/articles/:article_id/comments", () => {
```
Comment (yalicee) [inline __tests__/app.test.js]: 201 not 200

File: __tests__/app.test.js [hunk 2]
```diff
@@ -173,9 +173,120 @@ describe("200 - GET: /api/articles/:article_id/comments", () => {
...
+
+        expect(commentObj).toEqual({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: "butter_bridge",
+          body: "Your article is not very good you shouldnt be a journalist",
+          article_id: 1,
+        });
+      });
```
Comment (yalicee) [inline __tests__/app.test.js]: this is a successful 201? should be 404 right?

File: models/createComment.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  const username = req.username;
+  const body = req.body;
+  const commentQuery = `INSERT INTO comments (author, body, article_id) VALUES ($1, $2, $3) RETURNING *;`;
+  const articleQuery = format(
+    `SELECT * FROM articles WHERE article_id = %L;`,
+    articleID
+  );
+  return db.query(articleQuery).then((results) => {
+    if (results.rows.length === 0) {
+      const err = new Error("Article not found");
```
Comment (yalicee) [inline models/createComment.js]: article_id is also a foreign key so again, a separate check that the article exists should not be necessary as a psql error will be thrown

**Review by yalicee — CHANGES_REQUESTED** [review]
Looks great so far. See my comments :)

**Comment by steelfyi** [comment]
cooool think I've solved it hoepfully fingers crossed will send a new pr

**Review by yalicee — APPROVED** [review]
Loooks great fantastic work :) )))

**Pull request merged**